python/lsst/afw/table/source/source.cc

```diff
@@ -261,4 +245,4 @@ PYBIND11_PLUGIN(_source) {
     return mod.ptr();
 }
 
-}}}  // namespace lsst::afw::table
+}}}}  // namespace lsst::afw::table::<anonymous>
```

Comment:
I didn't think we had a convention for this, but I believe we did confirm that it doesn't matter in terms of code correctness (the `PYBIND11_PLUGIN` macro's `extern "C"` negates any namespaces it is defined in).